discretize/utils/meshutils.py

```diff
@@ -173,17 +173,16 @@ def closestPoints(mesh, pts, gridLoc='CC'):
 
 
 def ExtractCoreMesh(xyzlim, mesh, meshType='tensor'):
-    """Extracts Core Mesh from Global mesh
+    """
+    Extracts Core Mesh from Global mesh
 
     :param numpy.ndarray xyzlim: 2D array [ndim x 2]
     :param BaseMesh mesh: The mesh
 
     This function ouputs::
 
         - actind: corresponding boolean index from global to core
-        - meshcore: core mesh
-
-    Warning: 1D and 2D has not been tested
+        - meshcore: core sdiscretize mesh
     """
     import discretize
```

Comment:
Examples and tests sound awesome.

I think we might get into some import issues if we take `discretize` outside of the function. It is definitely better practice, though. I think ideally this would be a method on the mesh?

```python
core_mesh = mesh.core_mesh(lims)
```

I think for now just adding the tests would be good enough. Maybe an `assert` statement that doesn't let you do anything else other than `meshType='tensor'`? There is some of this started on for the `TreeMesh` on an issue. 

See #66.